msteams-platform/m365-apps/overview.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Extend Teams apps across Microsoft 365 (preview)
-description: In this module, know how to build and update your Teams app experiences across other high-usage areas of Microsoft 365. 
+description: In this article, learn how to build, update, and extend your Teams app experiences and how to use the Teams app manifest across other high-usage areas of Microsoft 365. 
```

Comment:
We are not using the app manifest across various surfaces. This edit is incorrect. 
End-users are using apps across surfaces. Manifest just helps create such apps.

Reply:
Edited and updated the description.